main.tf

```diff
@@ -196,6 +196,7 @@ resource "aws_s3_bucket" "this" {
         for_each = length(keys(lookup(server_side_encryption_configuration.value, "rule", {}))) == 0 ? [] : [lookup(server_side_encryption_configuration.value, "rule", {})]
 
         content {
+          bucket_key_enabled = lookup(rule.value, "bucket_key_enabled", null)
```

Comment:
Please do a similar update to the `object` submodule also.

Reply:
Sorry, I misread the contents of the first review.
I'll do it.